sandbox/src/mocks/recordTriageOutcome/responses/AcceptReferBookLater.json

```diff
@@ -42,6 +42,31 @@
         "system": "https://fhir.nhs.uk/Id/ods-organization-code",
         "value": "R69"
       }
+    },
+    {
+	"url": "https://fhir.nhs.uk/STU3/StructureDefinition/Extension-eRS-ReferralState-1",
+	"extension": [
+		{
+			"url": "state",
+			"valueCoding": {
+				"system": "https://fhir.nhs.uk/STU3/CodeSystem/eRS-ReferralState-1",
+				"code": "NOT_BOOKED",
+				"display": "Not Booked"
+			}
```

Comment:
Would the `NOT_BOOKED` state have a `reason` and `comment` associated? Should this referral not be in a `TRIAGE_RESPONSE` state?